conftest.py

```diff
@@ -1,7 +1,9 @@
 from typing import Dict, Tuple
+
```

Comment:
I suppose these line additions are a result of the initial `pre-commit` lint changes.

Reply:
Yes, black adds a line between standard and third-party library imports